src/common/Select.jsx

```diff
@@ -18,7 +18,7 @@ const Select = props => (
       value={props.value}
     >
 
-      <option value={props.placeholder}>{props.placeholder}</option>
+      <option value={props.value || props.placeholder}>{props.value || props.placeholder}</option>
```

Comment:
Consider destructuring the props above so you don't need to repeat the `props.value` i.e `let {value, placeholder, options} = this.props`